packages/api-client/src/types/index.ts

```diff
@@ -45,6 +45,7 @@ export type ApiConfig = {
   auth: AuthIntegrationContext;
   internationalization: InternationalizationIntegrationContext;
   backendUrl: string;
+  useFullImageUrls: boolean,
```

Comment:
This would make sense to be moved inside spreeFeatures, this way the user will be able to simply pick Vendo as a set of spree features to be enabled

Reply:
Well yeah, totally makes sense to me. Good practice, changed accordingly.